scripts/provisioning-tests

```diff
@@ -3,11 +3,15 @@ set -e
 
 cd $(dirname $0)/..
 
+eval "$(grep '^ENV CATTLE_SYSTEM_AGENT' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
+
 echo Starting rancher server
 mkdir -p /var/lib/rancher/k3s/agent/images
 grep PodTestImage ./tests/integration/pkg/defaults/defaults.go | cut -f2 -d'"' > /var/lib/rancher/k3s/agent/images/pull.txt
 grep MachineProvisionImage ./pkg/settings/setting.go | cut -f4 -d'"' >> /var/lib/rancher/k3s/agent/images/pull.txt
 touch /tmp/rancher.log
+mkdir -p /usr/share/rancher/ui/assets
+curl -sLf https://github.com/rancher/system-agent/releases/download/${CATTLE_SYSTEM_AGENT_VERSION}/rancher-system-agent-amd64 -o /usr/share/rancher/ui/assets/rancher-system-agent-amd64
```

Comment:
whats point of throwing them in this dir ?

Reply:
That's where system-agent is downloaded from by the downstream nodes. When running locally I needed to download binary there too (well, actually needed to change that path here https://github.com/rancher/rancher/blob/4dd4e3760cda7077de944061fd7187910c1334bd/pkg/settings/setting.go#L81 since `/usr/share` is not easily modifiable on MacOS so changed it to `/usr/local/share/rancher/ui`)